### PR TITLE
Deliverable 1

### PR DESCRIPTION
**Pull request by SophiaElizabeth**
First page of Monday issue

### CODE REVIEW

File: 1907-02-04.xml [hunk 1]
```diff
@@ -0,0 +1,1501 @@
...
+                    </div>   <div type="advert" xml:id="deg-ad-bam02">
+                        <head>Asia Minor Steamship Co., Ltd.</head>
+                        <p>Regular weekly sailings to Cyprus and Coast of Syria. Steamer leaves Alexandria
+                            every Tuesday afternoon for Limassol and Larnaca, returning viâ Port Said. 1st
+                            Class Single Fare, Alex. to Cyprus £2</p> 
+                        <p>Sailings liable to alteration owing to Quarantine, etc.</p>
+                        <p>For further particulars apply: Manager, 3 St. Mark's-Street, Alexandria.
+                            <measure type="indexNo">28478-31-12-06</measure></p>
+                    
+                        <!-- need template for A & M Samuelson Bros. Furniture -->
```
Comment (whanley) [inline 1907-02-04.xml]: did you file an issue?

File: 1907-02-04.xml [hunk 2]
```diff
@@ -0,0 +1,1501 @@
...
+                                >19-1-907</measure></p>
+                    </div>   <div type="advert" xml:id="deg-ad-bam02">
+                        <head>Asia Minor Steamship Co., Ltd.</head>
+                        <p>Regular weekly sailings to Cyprus and Coast of Syria. Steamer leaves Alexandria
+                            every Tuesday afternoon for Limassol and Larnaca, returning viâ Port Said. 1st
+                            Class Single Fare, Alex. to Cyprus £2</p> 
+                        <p>Sailings liable to alteration owing to Quarantine, etc.</p>
+                        <p>For further particulars apply: Manager, 3 St. Mark's-Street, Alexandria.
+                            <measure type="indexNo">28478-31-12-06</measure></p>
+
```
Comment (whanley) [inline 1907-02-04.xml]: You need to close this ad with a </div> tag

File: 1907-02-04.xml [hunk 3]
```diff
@@ -0,0 +1,1501 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://raw.githubusercontent.com/dig-eg-gaz/resources/master/out/egSchema.rnc" type="application/relax-ng-compact-syntax"?>
+<?xml-stylesheet type="text/xsl" href="dig-eg-gaz.xsl"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title level="m" type="main">Digital Egyptian Gazette</title>
+                <title level="m" type="sub">An encoded transcription</title>
+                <editor>Sophia Metallo= </editor>
```
Comment (whanley) [inline 1907-02-04.xml]: remove the =

File: 1907-02-04.xml [hunk 4]
```diff
@@ -0,0 +1,1501 @@
...
+                    <date when="2018-01-17">January 17, 2018></date>
+                </edition>
+            </editionStmt>
+            <publicationStmt>
+                <publisher>FSU University Libraries</publisher>
+                <pubPlace>Tallahassee, FL</pubPlace>
+            </publicationStmt>
+            <sourceDesc>
+                <bibl>
+                    <title>The Egyptian Gazette</title>
```
Comment (whanley) [inline 1907-02-04.xml]: remove ">February 4, 1907"

File: 1907-02-04.xml [hunk 5]
```diff
@@ -0,0 +1,1501 @@
...
+            </sourceDesc>
+        </fileDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="page" n="1">
+                <div type="masthead">
+                    <head>THE EGYPTIAN GAZETTE</head>
+                    <head>7678, FEBRUARY 4, 1907, EIGHT PAGES</head>
+                    <head type="sub">No. <measure quantity="7239">7,239</measure>] ALEXANDRIA, MONDAY, FEBRUARY 4, 1907. [EIGHT PAGES P.T.
```
Comment (whanley) [inline 1907-02-04.xml]: proper issue number?

File: 1907-02-04.xml (hunk 1 above)
Comment (SophiaElizabeth) [reply 1907-02-04.xml]: Just filed issue

**Comment by whanley** [comment]
Some things fixed--thanks--but dates still not corrected.

**Comment by whanley** [comment]
Still lots of updates needed for deliverable 1 (page 1).

File: 1907-02-04.xml [hunk 6]
```diff
@@ -0,0 +1,2451 @@
...
+                </bibl>
+            </sourceDesc>
+        </fileDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="page" n="1">
+                <div type="masthead">
+                    <head>THE EGYPTIAN GAZETTE</head>
+                    <head>7678, FEBRUARY 4, 1907, EIGHT PAGES</head>
```
Comment (whanley) [inline 1907-02-04.xml]: delete this line

File: 1907-02-04.xml [hunk 7]
```diff
@@ -0,0 +1,2451 @@
...
+                    <head type="sub">No. 7678 <measure quantity="7239">7,239</measure>] ALEXANDRIA,
```
Comment (whanley) [inline 1907-02-04.xml]: This line should read `<head type="sub">No. <measure quantity="7678">7,678</measure>] ALEXANDRIA,`

File: 1907-02-04.xml [hunk 8]
```diff
@@ -0,0 +1,2451 @@
...
+                    </div>
+                    <div type="advert" xml:id="deg-ad-bam02">
+                        <head>Asia Minor Steamship Co., Ltd.</head>
+                        <p>Regular weekly sailings to Cyprus and Coast of Syria. Steamer leaves
+                            Alexandria every Tuesday afternoon for Limassol and Larnaca, returning
+                            viâ Port Said. 1st Class Single Fare, Alex. to Cyprus £2</p>
+                        <p>Sailings liable to alteration owing to Quarantine, etc.</p>
+                        <p>For further particulars apply: Manager, 3 St. Mark's-Street, Alexandria.
+                                <measure type="indexNo">28478-31-12-06</measure></p>
+
```
Comment (whanley) [inline 1907-02-04.xml]: Insert a `</div>` tag to close this ad

File: 1907-02-04.xml [hunk 9]
```diff
@@ -0,0 +1,2451 @@
...
+                                    <cell>,, ,,</cell>
+                                    <cell><date when="1907-03-02">2, 9, 16, 23, 30 March</date>
+                                        ,,</cell>
+                                    <cell>,, ,,</cell>
+                                    <cell><date when="1907-03-05">5, 12, 19, 26 March</date>
+                                        ,,</cell>
+                                </row>
+                                <row>
+                                    <cell>,, ,,</cell>
+                                    <cell><date when="1907-04-
```
Comment (whanley) [inline 1907-02-04.xml]: most of these should be available now

Comment (whanley) [inline 1907-02-04.xml]: insert `</div>` to close this ad.

Comment (whanley) [inline 1907-02-04.xml]: insert `</div>` tag to close previous item.

**Comment by whanley** [comment]
Good. A bit more work to do, but you are almost complete deliverable 2, and not too far with 3 either.

- There are more ad templates posted all the time. Check again to see if I've added what was missing.
- line 1673: probably you have an extra `</div>` somewhere that needs to be closed? Probably at the end of page 2, where you have three in a row.
- line 1740: you need to make "Mortgages in Egypt" a separate div.

Page 3:
- local and general section needs to be formatted differently--every item its own div (inside the section), with its own headline.
- 2122 missing headline?
- 2232: not element="local"
- 2293: not element="legal"

Page 4:
- 2576 add element
- 3037 cheap prepaid advertisements section should end here

Page 5:
 are you missing material?

Page 6:
- 3433 close div
- marche de minet el bassal needs to be updated?
5697 one too many `</div>`s?

Page 7 and 8.
These are new to me. Please upload images of these pages and I'll see what I can do.

**Comment by whanley** [comment]
Monday issue:
- still need to incorporate my previous comments, starting on page 3. As I said elsewhere, upload your page images and I'll be able to look at pages 7 and 8.

Tuesday: good start, keep at it.

**Comment by whanley** [comment]
The blog post has to go in a different repository. Check the how-to.

**Comment by whanley** [comment]
Monday issue:
- deliverable 2 complete
- deliverable 3 complete, but please fix these things:
- page 4: put wire reports _inside_ telegrams section in column one--you have it right in column 2.
- page 6: line 3510: close Bulletin div.

Tuesday issue: complete, but there's still tidying to do in weeks to come as I post more templates.

Wednesday issue: please work on pages 4, 6: there are double headlines that you break into separate divs. One div per article please!

Remove the  Church for Tourists (Switzerland).jpg image from your repository.

**Pull request merged**